osrf/vrx
# Move 2019 assets to vrx_2019

See #298.

This pull request moves all the 2019 world and launch files to a new `vrx_2019` package. I also restored some of the 2019 world files that were updated to the 2022 rules.

How to test it:

* I suggest removing the `build` and `devel` directories and recompile.
* Then, you can try launching all the worlds in the new package. Example:

```
roslaunch vrx_2019 vrx.launch
```

and confirm that the world loads correctly.
